infra/cifuzz/cifuzz_test.py

```diff
@@ -28,9 +29,16 @@
 import fuzz_target
 
 # NOTE: This integration test relies on
-# https://github.com/google/oss-fuzz/tree/master/projects/example project
+# https://github.com/google/oss-fuzz/tree/master/projects/example project.
 EXAMPLE_PROJECT = 'example'
 
+# Location of files used for testing.
+TEST_FILES_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                               'test_files')
+
+# An Example fuzzer that trips an error.
```

Comment:
super nit: maybe `triggers` instead of `trips`

Reply:
Another nit: "example" not "Example"

Reply:
Fixed to both.